### ISSUE TITLE
lock throws NullPointerException

### ISSUE THREAD

**Issue by fkorotkov** [gogradle/gogradle#183]
I've tried to update my lock file after some time by simply re-generating it(as I did it before) and got:

```
Caused by: java.lang.NullPointerException
        at com.github.blindpirate.gogradle.util.StringUtils.toUnixString(StringUtils.java:92)
        at com.github.blindpirate.gogradle.core.dependency.LocalDirectoryDependency.toLockedNotation(LocalDirectoryDependency.java:100)
        at com.github.blindpirate.gogradle.core.dependency.lock.DefaultLockedDependencyManager.toNotations(DefaultLockedDependencyManager.java:69)
        at com.github.blindpirate.gogradle.core.dependency.lock.DefaultLockedDependencyManager.lock(DefaultLockedDependencyManager.java:53)
        at com.github.blindpirate.gogradle.task.GoLock.lock(GoLock.java:50)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
```

Not sure which dependency causes it since there not much logs.

**Comment by blindpirate** [comment]
Thanks for reporting, would you like to share your `build.gradle` if possible?

**Comment by fkorotkov** [comment]
Sure! Here is is:

```
apply plugin: "com.github.blindpirate.gogradle"

golang {
  goVersion = '1.9.2'
}

golang {
  packagePath = 'github.com/example/agent'
}

build {
  targetPlatform = ['windows-amd64', 'linux-amd64', 'darwin-amd64']
  environment 'CGO_ENABLED', '0' // to not have a dep on system libraries
}

repositories {
  golang {
    // seems gogradle doesn't like it
    root { it.startsWith('github.com/apache/thrift') }
    emptyDir()
  }
}

dependencies {
  golang {
    build 'github.com/certifi/gocertifi@2017.11.05'
    build 'github.com/golang/protobuf/proto'
    build 'github.com/grpc-ecosystem/go-grpc-middleware/retry'
    build 'github.com/src-d/go-git@v4.0.0-rc15'
    build 'golang.org/x/net/context'
    build 'google.golang.org/grpc@v1.7.2'
  }
}

idea {
  module {
    generatedSourceDirs += file(".gogradle/project_gopath/src")
  }
}
```

**Comment by blindpirate** [comment]
Fixed by `0.8.1`.
